ui/src/ScholarReader.tsx

```diff
@@ -726,14 +727,27 @@ export default class ScholarReader extends React.PureComponent<Props, State> {
     });
   }
 
+  parseQueryString = (): qs.ParsedQs => {
```

Comment:
nit pick: `index.tsx` file uses `query-string` library for similar functionality. Maybe we can use just one of these libraries or the other? https://github.com/allenai/scholarphi/blob/95ad49a46e914863d2873f19c2b9875bf4bbe94f/ui/src/index.tsx#L1

Reply:
Somehow I missed that in the dependencies! I'll switch to using that since it does the same job and includes its own types.